src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -311,6 +351,31 @@ public void openBrowserAndConnect(String serviceProtocolUri, String page) {
       null
     );
   }
+
+  public void openPanel(String serviceProtocolUri, ContentManager contentManager, String tabName) {
+    String address = DevToolsUtils.generateDevToolsUrl(devtoolsHost, devtoolsPort, serviceProtocolUri, null) + "&embed=true&page=inspector";
+
+    EngineOptions options =
+      EngineOptions.newBuilder(HARDWARE_ACCELERATED).build();
+    Engine engine = Engine.newInstance(options);
+    Browser browser = engine.newBrowser();
+
+    final Content content = contentManager.getFactory().createContent(null, tabName, false);
+
+    SwingUtilities.invokeLater(() -> {
+      // Creating Swing component for rendering web content
+      // loaded in the given Browser instance.
+      BrowserView view = BrowserView.newInstance(browser);
+      view.setPreferredSize(new Dimension(contentManager.getComponent().getWidth(), contentManager.getComponent().getHeight()));
+
+      content.setComponent(view);
+      content.putUserData(ToolWindow.SHOW_CONTENT_ICON, Boolean.TRUE);
+      content.setIcon(FlutterIcons.Phone);
```

Comment:
add a todo: we should use a different icon based on the tab and we should copy the icons from the inspector toolbar rather than using the Phone icon.